pydis_site/static/css/content/page.css

```diff
@@ -68,3 +68,12 @@ ul.menu-list.toc {
 .toc > ul > li {
     list-style-type: disc;
 }
+
+/*
+ * Increase space between images and any text above the image in a list.
+ * This is used instead of a newline between the image and the text in markdown
+ * so the image remains left-aligned with the list item.
+ */
+li img {
```

Comment:
Why not do this for both ordered and unordered lists?

Reply:
Ordered and unordered lists both use the `<li>` tag for list items, which is what I'm styling here, so it works for both.